src/ActionInputValidator/ValidatorFactory.ts

```diff
@@ -43,7 +43,7 @@ export class ValidatorFactory {
             }
         }
         else {
-            throw new Error("Valid credentails are not available. Add Azure Login action before this action or provide publish-profile input.");

```

Comment:
Can you merge latest master to your fork and run `npm run build`?
Please add lib folder changes as well in this PR

Reply:
I've updated the file in `lib`, but running the build after an `npm install` generated more changes that just the typo fix, so I've not committed that as that seems beyond the scope of this typo fix as it changed various dependencies in the lock file.